src/database/standalone-sqlite.ts

```
@@ -4096,6 +4124,10 @@ if (!isMainThread) {
           const tx = worker.getTxByOffset(args[0]);
           parentPort?.postMessage(tx);
           break;
+        case 'getBlockByWeaveOffset':
+          const blockByWeaveOffset = worker.getBlockByWeaveOffset(args[0]);
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)